package.json

```diff
@@ -19,9 +19,27 @@
     "atom": ">=0.174.0 <2.0.0"
   },
   "dependencies": {
-    "atom-space-pen-views": "^2.0.3"
+    "atom-space-pen-views": "^2.0.3",
+    "atom-ts-transpiler": "^1.2.3",
+    "typescript": "^2.5.3"
   },
   "devDependencies": {
     "coffeelint": "1.16.0"
-  }
+  },
+  "atomTranspilers": [
+        {
+            "transpiler": "atom-ts-transpiler",
+            "glob": "{!(node_modules)/**/,}*.ts?(x)",
+            "options": {
+                "cacheKeyFiles": [
+                    "lib/tsconfig.json"
+                ],
+                "verbose": false
+            }
+        },
+        {
+            "glob": "{!(node_modules)/**/,}*.js",
+            "transpiler": "atom-babel6-transpiler"
+        }
```

Comment:
Well, now I feel dumb. Took me about 30 seconds to realize what's going on after actually taking a serious look at the changes. Atom's tripping up because you have `atom-babel6-transpiler` listed as a transpilation handler, but it's not in the package's dependencies, so trying to register the transpiler fails (meaning `apm test` exits with code 1 before ever trying to run the specs.)

In fact, I would just remove this entry altogether, since having a custom transpiler for .js files isn't actually necessary here, and the glob doesn't match any files, anyway. I might also point out that this section is using 4-space indentation whereas the rest of the file is using 2-space indentation, although that's less of a serious issue and more of a style thing.